README.md

```diff
@@ -1,8 +1,10 @@
-# CloudPak Community Automation
+# Redhat Openshift Container (OCP) Community Automation
 
 ## Introduction
 
-This repo represents the Community Automation effort where teams can contribute automation to be shared with other teams.  We decided as a guild to use Jenkins and Ansible combination for our implementations.  Jenkins and Ansible details below.  The repo is also setup for use as an ansible collection to be included in playbooks being created outside of this repo.
+This repo represents the Redhat Openshift Container Community Automation effort where teams can contribute OCP automation to be shared with other teams.  Iwas decided early on in the project to use Jenkins and Ansible combination for our implementations.  Jenkins and Ansible details below.  The repo is also setup for use as an ansible collection to be included in playbooks being created outside of this repo.  
```

Comment:
Same as above

This repo represents the Red Hat OpenShift Container Platform Community Automation ...